modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -49,6 +49,21 @@ This resulted in an inconsistent end-user experience.
 
 For information on the **Merge Tags** plugin, see: xref:mergetags.adoc[Merge Tags].
 
+=== Footnotes 1.0.1
+
+The {productname} 6.5 release includes an accompanying release of the **Footnotes** premium plugin.
+
+**Footnotes** 1.0.1 includes one fix:
```

Comment:
Should this be here the ticket is for templates the ticket had the wrong component.

Reply:
@spocke Updated, thanks for the heads up.